pkgs/build-support/php/pkgs/composer-phar.nix

```diff
@@ -0,0 +1,48 @@
+{
+  _7zz
+  , cacert
+  , curl
+  , fetchurl
+  , git
+  , lib
+  , makeBinaryWrapper
+  , php
+  , stdenvNoCC
+  , unzip
+  , xz
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "composer-phar";
+  version = "2.6.2";
+
+  src = fetchurl {
+    url = "https://github.com/composer/composer/releases/download/${finalAttrs.version}/composer.phar";
+    hash = "sha256-iMhNSlP88cJ9Z2Lh1da3DVfG3J0uIxT9Cdv4a/YeGu8=";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin
+    install -D $src $out/libexec/composer/composer.phar
+    makeWrapper ${php}/bin/php $out/bin/composer \
+      --add-flags "$out/libexec/composer/composer.phar" \
+      --prefix PATH : ${lib.makeBinPath [ _7zz cacert curl git unzip xz ]}
+
+    runHook postInstall
+  '';
+
+  meta = {
+    changelog = "https://github.com/composer/composer/releases/tag/${finalAttrs.version}";
+    description = "Dependency Manager for PHP, shipped from the PHAR file";
+    homepage = "https://getcomposer.org/";
+    license = lib.licenses.mit;
+    maintainers = lib.teams.php.members;
```

Comment:
This is the last example where it's not obvious to me if it existed before?

Reply:
This was existing before, @etu just moved the file.

Reply:
This would also mean that we lost the codeowners of this file :wink: So interests for owning this file may change as well since it serves a different purpose now compared to before.

But yes, I can confirm that this is the vendored composer which is exactly the one we ship as primary composer today, however it will only be used as a bootstrap for a composer built from source in the future so we can build more packages from source as well.

Reply:
@drupol I think we maybe should remove the team here as well since the context has switched for this package?

Reply:
Done.